### ISSUE TITLE
Problem with block as a parameter in Swift

### ISSUE THREAD

**Issue by tosbaha** [onevcat/VVDocumenter-Xcode#118]
```swift
    func getSomething(input:String?, success: (Int) -> Void, failure: (NSHTTPURLResponse?, AnyObject?, NSError?) -> Void) {

    }
```
By using this function, plugin creates following documentation which lacks ```failure``` parameter

    /**
    <#Description#>
    
    :param: input   <#input description#>
    :param: success <#success description#>
    */

**Comment by onevcat** [comment]
Thanks for the feedback. I will take a look on it soon.

**Comment by onevcat** [comment]
Should be fixed in #125
